### ISSUE TITLE
Usage of imp causes deprecation warnings

### ISSUE THREAD

**Issue by wimglenn** [pypa/distlib#169]
Using `distlib.wheel` now gets warnings

```
.venv/lib/python3.10/site-packages/distlib/wheel.py:14
  .../.venv/lib/python3.10/site-packages/distlib/wheel.py:14: DeprecationWarning: the imp module is deprecated in favour of importlib and slated for removal in Python 3.12; see the module's documentation for alternative uses
```

**Comment by vsajip** [comment]
Unfortunately there's no replacement for `imp.load_dynamic()`, or at least that's my understanding.

**Comment by wimglenn** [comment]
What's distlib's plan for Python 3.12, in that case?

**Comment by vsajip** [comment]
To look at replacing it with something on top of `importlib`.

**Comment by vsajip** [comment]
Should be fixed in 3be4cc0.
